src/components/NestedAccordionMenu/MenuItem.js

```diff
@@ -0,0 +1,102 @@
+import React, { Fragment, useState } from "react";
+import List from "@material-ui/core/List";
+import ListItem from "@material-ui/core/ListItem";
+import ExpandMoreIcon from "@material-ui/icons/ExpandMore";
+import ExpandLessIcon from "@material-ui/icons/ExpandLess";
+import Collapse from "@material-ui/core/Collapse";
+import ListItemIcon from "@material-ui/core/ListItemIcon";
+import ListItemText from "@material-ui/core/ListItemText";
+import { makeStyles } from "@material-ui/core/styles";
+const useStyles = makeStyles(theme => ({
+  sidebarItem: {
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+  },
+  sidebarItemContent: {
+    whiteSpace: "nowrap",
+    textOverflow: "ellipsis",
+    overflow: "hidden",
+    display: "flex",
+    alignItems: "center",
+    width: "100%",
+  },
+  collapse: {
+    background: "#f4f1f1",
+  },
+}));
+function MenuItem({ depthStep = 10, depth = 0, expanded, item, ...rest }) {
+  const classes = useStyles();
+  const [collapsed, setCollapsed] = useState(true);
+  const { route, name, items, Icon } = item;
+
+  function toggleCollapse() {
+    setCollapsed(prevValue => !prevValue);
+  }
+
+  function onClick(e) {
+    if (Array.isArray(items)) {
+      toggleCollapse();
+    }
+    if (onClickProp) {
+      onClickProp(e, item);
+    }
+  }
+
+  let expandIcon;
+
+  if (Array.isArray(items) && items.length) {
+    expandIcon = !collapsed ? (
+      <ExpandLessIcon className={classes.sidebarItemExpandArrowAxpanded} />
+    ) : (
+      <ExpandMoreIcon className={classes.sidebarItemExpandArrow} />
+    );
+  }
+
+  return (
+    <Fragment>
+      <ListItem
+        className={classes.sidebarItem}
+        onClick={onClick}
+        button
+        dense
+        {...rest}
+        to={route}
+      >
+        <div
+          style={{ paddingLeft: depth * depthStep }}
+          className={classes.sidebarItemContent}
+        >
+          <ListItemIcon>
+            {Icon && <Icon className={classes.sidebarItemIcon} />}
+          </ListItemIcon>
+          <ListItemText primary={name} />
+        </div>
+        {expandIcon}
+      </ListItem>
+      <Collapse
+        className={classes.collapse}
```

Comment:
Only the background of the nested list should be dark. The parent item of the first level should stay white

Reply:
is this background: "#d7d9db" match what should look like the nested list?
![Nested Accordion Menu   Default ⋅ Storybook(1)](https://user-images.githubusercontent.com/19631540/69614288-8d6bf980-103b-11ea-8c8c-ee666cf8116b.png)

Reply:
The nested list background should be #DCDBDC as per the D2D screenshot.
And current highlighted nested item should not have a different background but text color should use the theme primary color.
But If the submenu is closed but a submenu item is highlighted, then the parent should have the same highlighted background as if it was a first level item without children (#EFEFEF).

So highlight rules are:
- route match first level item with no children -> background #efefef
- route match submenu item and parent open -> parent no background - sub item no background and color primary
- route match submenu item and parent closed -> parent background #efefef